{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/models.py

```diff
@@ -76,6 +120,76 @@ class Career(PageBase):
         blank=True,
     )
 
+    date_posted = models.DateField(
+        auto_now_add=True,
+        blank=True,
+        null=True,
+    )
+
+    # Schema Fields
+    employment_type = models.CharField(
+        choices=[
+            ('FULL_TIME', 'Full time'),
```

Comment:
Perhaps I'm missing something, but I don't see a reason that the database values (`choice[0]`) needs to differ from the displayed name (`choice[1]`) here. Unless the caps lock version is required by the schema?